pull_request_template.md

```diff
@@ -1,13 +1,35 @@
-#### What's this PR do?
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
I realized this PR template isn't descriptive enough - which is understandable, since I first began using it three years ago when I was a bootcamp student. I decided to update it to a different PR template that I prefer.